pkg/app/piped/eventwatcher/BUILD.bazel

```diff
@@ -6,14 +6,12 @@ go_library(
     importpath = "github.com/pipe-cd/pipecd/pkg/app/piped/eventwatcher",
     visibility = ["//visibility:public"],
     deps = [
-        "//pkg/app/server/service/pipedservice:go_default_library",
         "//pkg/backoff:go_default_library",
         "//pkg/config:go_default_library",
         "//pkg/git:go_default_library",
         "//pkg/model:go_default_library",
         "//pkg/regexpool:go_default_library",
         "//pkg/yamlprocessor:go_default_library",
-        "@org_golang_google_grpc//:go_default_library",
```

Comment:
I think this change does not relate to this PR. Besides, looks like it related to nakabonne's pull https://github.com/pipe-cd/pipecd/pull/3147#issuecomment-1023819321

Reply:
Thanks. Removed it.